cypress/e2e/tests/migration/applicationinventory/analysis/source_analysis.test.ts

```diff
@@ -241,6 +241,24 @@ describe(["@tier1"], "Source Analysis", () => {
         application.verifyEffort(2);
     });
 
+    it("JWS6 target Source + deps analysis on tackletest app", function () {
+        // Source code analysis require both source and maven credentials
+        const application = new Analysis(
+            getRandomApplicationData("tackleTestApp_Source+dependencies_jws6", {
+                sourceData: this.appData["tackle-testapp-git"],
+            }),
+            getRandomAnalysisData(this.analysisData["jws6_source+dep_analysis_on_tackletestapp"])
+        );
+        application.create();
+        applicationsList.push(application);
+        cy.wait("@getApplication");
+        cy.wait(2 * SEC);
+        application.manageCredentials(source_credential.name, maven_credential.name);
+        application.analyze();
+        application.verifyAnalysisStatus("Completed");
+        application.verifyEffort(3);
```

Comment:
effort can also be added to anaysis.json as a parameter.

Reply:
Tried to accept the effort parameter from analysis.json but returning "undefined", I guess need to debug and fix the verifyEffort() method in another PR. For now effort in hardcoded form we can keep.
![Screenshot from 2024-02-13 15-18-34](https://github.com/konveyor/tackle-ui-tests/assets/87791967/c7418844-3b88-4392-9e70-8c5254e4e1af)
![Screenshot from 2024-02-13 16-58-39](https://github.com/konveyor/tackle-ui-tests/assets/87791967/77fa586f-0d89-4e2c-bb6a-1af657eca848)